spec/requests/schools_request_spec.rb

```diff
@@ -88,7 +88,7 @@
     end
 
     [
-      [2020, 2, 30],
+      [2020, 2, 32],
```

Comment:
30th of February should be invalid date too, right?

Reply:
Rails tries to be clever and automatically changes it to 1st March.